torch/onnx/symbolic_opset9.py

```diff
@@ -1919,3 +1919,12 @@ def multinomial(g, input, num_samples, replacement=False, generator=None):
     return g.op("Multinomial", log_input,
                 dtype_i=sym_help.cast_pytorch_to_onnx['Long'],
                 sample_size_i=num_samples)
+
+
+@parse_args('v', 'v', 'v', 'v', 'v')
+def baddbmm(g, self, batch1, batch2, beta, alpha):
```

Comment:
If everything is values, what's the point of using parse_args?

Also beta and alpha could be scalar.

Reply:
@houseroad I updates this. Thanks.